SKILL.tsv

```diff
@@ -5359,29 +5359,29 @@ SKILL_20151022_005358
 SKILL_20151022_005359	
 SKILL_20151022_005360	
 SKILL_20151022_005361	$Attaque Composee
-SKILL_20151022_005362	
+SKILL_20151022_005362	${#DD5500}{ol}[Physical] - [Pierce]{/}{/}Stab an enemy in front of you.
 SKILL_20151022_005363	$Lunge
-SKILL_20151022_005364	
-SKILL_20151022_005365	
+SKILL_20151022_005364	${#DD5500}{ol}[Physical] - [Pierce]{/}{/}Lower your stance and reach out to attack an enemy. Your evasion rate increases and the enemy becomes more vulnerable to [Slash] attacks.
+SKILL_20151022_005365	$Attack #{SkillAtkAdd}#{nl}Attribute Damage #{CaptionRatio}#%{nl}Evasion Rate #{CaptionRatio2}#%{nl}Duration #{CaptionTime}# seconds
 SKILL_20151022_005366	$Sept Etoiles
-SKILL_20151022_005367	
+SKILL_20151022_005367	${#DD5500}{ol}[Physical] - [Pierce]{/}{/}Stab an enemy with consecutive strikes.
 SKILL_20151022_005368	$Attaque Coquille
-SKILL_20151022_005369	
+SKILL_20151022_005369	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}Attack an enemy with the guard of your sword. The enemy's defence is ignored when attacked again with [Pierce] attacks.
 SKILL_20151022_005370	$Esquive Toucher	
-SKILL_20151022_005371	
-SKILL_20151022_005372	
+SKILL_20151022_005371	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}Move swiftly to avoid enemy attacks and deliver complex consecutive attacks.
+SKILL_20151022_005372	$Attack #{SkillAtkAdd}#{nl}Attribute Damage #{CaptionRatio}#%{nl}Evasion Rate #{CaptionRatio2}#%
 SKILL_20151022_005373	$Preparation
-SKILL_20151022_005374	
-SKILL_20151022_005375	
+SKILL_20151022_005374	$Twirl the tip of your sword to parry an enemy attack and prepare for the next attack. The next [Pierce] attack you make will inflict twice as much damage.
+SKILL_20151022_005375	$Max Duration #{CaptionRatio2}#seconds{nl}Block Rate #{CaptionRatio}#%{nl}Duration 15 seconds
```

Comment:
Max Duration #{CaptionRatio2}#seconds{nl}Block Rate #{CaptionRatio}#%{nl}Duration 15 seconds

to

Maximum Duration: #{CaptionRatio2}# seconds{nl}Block Rate +#{CaptionRatio}#%{nl}Duration: 15 seconds